integreat_cms/cms/forms/pages/page_filter_form.py

```diff
@@ -26,7 +50,6 @@ class PageFilterForm(CustomFilterForm):
     def __init__(self, **kwargs):
         r"""
         Initialize page filter form
-
         :param \**kwargs: The supplied keyword arguments
```

Comment:
```suggestion

        :param \**kwargs: The supplied keyword arguments
```
I think a separation line between the docstring and the param documentation makes it more readable. No need to remove the line.